CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 set(Boost_USE_STATIC_LIBS ON)
 
-set(FDB_VERSION 6.0.11)
+set(FDB_VERSION release-6.0)
```

Comment:
We can depend on `6.0.17`, instead. It has the fix we care about- https://github.com/apple/foundationdb/commit/776b289bfe3379fb517b5bcf693c0959b0997a8e

Reply:
Sure thing.